### DISCUSSION TITLE
Težava z gumbom na spletni strani

### DISCUSSION THREAD

**Discussion by bojana202** [jurerotar/Digital-communications-demo#22] [General]
Pozdravljeni,
razvijam spletno stran(hotela bi dodati še eno lekcijo iz učbenika) in naletela sem na težavo pri uporabi gumba. Ko kliknem na gumb, ki je namenjen preusmeritvi na novo lekcijo, se nič ne zgodi in uporabnik ni preusmerjen na novo stran.Ali bi lahko preverili, ali je mogoče kaj narobe s povezavo ali z nastavitvami za preusmerjanje?
Hvala vnaprej za pomoč!

**Comment by jurerotar** [comment]
Lep pozdrav!

O katerem gumbu specifično govorimo? Sem na hitro poklikal aplikacijo, nisem opazil težav pri navigaciji čez stran. Prosim za posnetek/screenshot in pa informacije o okolju (torej, ali se težava pojavi pri lokalnem razvoju, ali je prisotna tudi na produkciji, informacije o brskalniku,...).

**Comment by bojana202** [comment]
Najlepšaa hiter odgovor!Hotela sem na obstoječo spletno stran dodati razlago za še eno lekcijo (Viterbi algoritem) in sem dodala gumb za to lekcijo. Težava je v tem, da ko se klika na novo lekcijo Viterbi algoritem me ne preusmeri na to lekcijo/spleetno stran.
V čakanju na vaš odgovor vas lepo pozdravljam,Bojana Bogičević
    On Tuesday, December 31, 2024 at 07:17:20 PM GMT+1, Jure Rotar ***@***.***> wrote:  
 
 


Lep pozdrav!

O katerem gumbu specifično govorimo? Sem na hitro poklikal aplikacijo, nisem opazil težav pri navigaciji čez stran. Prosim za posnetek/screenshot in pa informacije o okolju (torej, ali se težava pojavi pri lokalnem razvoju, ali je prisotna tudi na produkciji, informacije o brskalniku,...).

**Comment by jurerotar** [comment]
Težko pomagam, brez da vidim kaj je bilo spremenjeno. Prosim, če odpreš pull request, in pogledam!

**Comment by jurerotar** [comment]
Ste uspeli rešiti težavo, ali še potrebujete pomoč? 😄

**Comment by bojana202** [comment]
Pozdravljeni,
Hvala za vaše vprašanje!  Začelo se je izpitno obdobje, zato sem pozornost preusmerila na izpitne obveznosti in spletno stran za zdaj pustila malo ob strani. Ko se zaključi izpitne obdoblje se bom vrnila spletni strani.Lep pozdrav,Bojana Bogičević

    On Friday, January 17, 2025 at 08:47:49 PM GMT+1, Jure Rotar ***@***.***> wrote:  
 
 


Ste uspeli rešiti težavo, ali še potrebujete pomoč? 😄

**Reply by jurerotar** [reply]
Vso srečo z izpiti!

**Comment by bojana202** [comment]
Najlepša hvala!
    On Friday, January 17, 2025 at 09:24:52 PM GMT+1, Jure Rotar ***@***.***> wrote:  
 
 


Vso srečo z izpiti!
